java-manta-client-unshaded/src/main/java/com/joyent/manta/config/BaseChainedConfigContext.java

```diff
@@ -391,6 +391,25 @@ public String getEncryptionPrivateKeyPath() {
         return encryptionPrivateKeyBytes;
     }
 
+    /**
+     * Accessor for field that tracks provenance of manta key path
+     * field, specifically set to `true` if the manta key path has
+     * only been set because a DefaultConfigContext was added to the
+     * chain of configs. It's important to preserve this information
+     * up the chain (for example if a StandardConfigContext is created
+     * with a DefaultConfigContext and then added to another chain, it
+     * is important to preserve that the manta key path was set
+     * originally (and only) by the DefaultConfigContext)
+     *
+     * @return boolean - true if mantaKeyPath came from
+     * DefaultConfigContext, false otherwise
+     * @see overwriteWithContext for implementation details
```

Comment:
Javadoc syntax is silly. This @see statement doesn't result in a javadoc link. Try this instead:
```
@see BaseChainedConfigContext#overwriteWithContext overwriteWithContext for implementation details
```